app/assess/data.py

```diff
@@ -377,6 +377,29 @@ def get_comments(application_id: str, sub_criteria_id: str):
     comment_response = get_data(comment_endpoint)
     return comment_response
 
+
+def submit_comment(
+    comment, application_id, sub_criteria_id, user_id, theme_id
+):
+    data_dict = {
+        "comment": comment,
+        "user_id": user_id,
+        "application_id": application_id,
+        "sub_criteria_id": sub_criteria_id,
+        "comment_type": "COMMENT",
+        "theme_id": theme_id
+    }
+    url = Config.ASSESSMENT_COMMENT_ENDPOINT
+    response = requests.post(url, json=data_dict)
+    current_app.logger.info(
+        f"Response from Assessment Store: '{response.json()}'."
+    )
+    if response.status_code == 200:
+        return True
+    else:
+        return False
```

Comment:
This won't work as you have the `status_code` set to `201` on your store PR: https://github.com/communitiesuk/funding-service-design-assessment-store/pull/69/files#diff-dbf73fdcc85b9741624a4d69562ff262127fe585b5b599f0e6f643f5634fc565R359-R367

Maybe could do this instead?
```suggestion
    return response.ok 
```

Reply:
Although I don't think the return value is actually being used, so not a big deal

Reply:
Amended to response.ok